DataConnectors/SonicWall/SonicwallFirewall.JSON

```diff
@@ -0,0 +1,118 @@
+{
+    "id": "SonicWallFirewall",
+    "title": "SonicWall Firewall",
+    "publisher": "SonicWall",
+    "descriptionMarkdown": "Common Event Format (CEF) is an industry standard format on top of Syslog messages, used by SonicWall to allow event interoperability among different platforms. By connecting your CEF logs to Azure Sentinel, you can take advantage of search & correlation, alerting, and threat intelligence enrichment for each log.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "SonicWall",
+            "baseQuery": "CommonSecurityLog\n| where DeviceVendor == \"SonicWall\"\n| where DeviceProduct == \"firewall\""
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "All logs",
+            "query": "\nCommonSecurityLog\n| where DeviceVendor == \"SonicWall\"\n| where DeviceProduct == \"firewall\"\n\n            | sort by TimeGenerated"
+        },
+        {
+            "description" : "Summarize by destination IP and port",
+            "query": "\nCommonSecurityLog\n| where DeviceVendor == \"SonicWall\"\n| where DeviceProduct == \"firewall\"\n\n            | summarize count() by DestinationIP, DestinationPort\n            | sort by TimeGenerated"

```

Comment:
This query has modified to have "count() by dst, proto\n            | sort by TimeGenerated" - There are no columns by name dst and proto in CommonSecurityLog. 
The earlier query has DestinationIP and DestinationPort - These 2 columns are correct, The problem is the summarize query return only 2 columns DestinationIP and DestinationPort. But the query has sorting by "TimeGenerated" which is not returned since result has only 2 columns DestinationIP  and DestinationPort, hence the results cannot be sorted by a column "TimeGenerated" which is not returned. 

If you want to have the results to be sorted by TimeGenerated, you have to include that column in summarize.
Working Sample Query that returns 3 columns DestinationIP, DestinationPort and TimeGenerated: 
CommonSecurityLog | where DeviceVendor == "SonicWall" | where DeviceProduct == "firewall"             
| summarize count() by DestinationIP, DestinationPort , TimeGenerated        
   | sort by TimeGenerated
-- Since TimeGenerated is returned, results can be sorted by TimeGenerated.